vendor/github.com/percona/exporter_shared/helpers/helpers.go

```diff
@@ -67,5 +67,8 @@ func ReadMetric(m prometheus.Metric) *Metric {
 	if pb.Untyped != nil {
 		return &Metric{name, help, labels, dto.MetricType_UNTYPED, pb.GetUntyped().GetValue()}
 	}
+	if pb.Histogram != nil {
```

Comment:
@svistoi Oh no... 😕  You need to add this to https://github.com/percona/exporter_shared You can create the same branch/PR name. I'll check and merge that thing in `exporter_shared`. After that, you can do `dep ensure -update github.com/percona/exporter_shared` here and I think it will work finally and we can merge it.

Reply:
Looks like exporter_shared dependency is set to v0.0.0-20180321061342-7d9e9cc0256a

helper.go was deleted in commit 37527a48217c926b72e13d08e35776abd3a3cdcb.  

I can branch from 7d9e9cc0256a and create a PR, but `dep ensure -update` to current HEAD will break?

Reply:
Also if you could please create a branch in exporter_shared for me to submit the merge request to.

Reply:
@svistoi Oh, ok let's try another way. I have made a PR (https://github.com/percona/exporter_shared/pull/5) and when it will be merged, I update this dependency in `master` and give you feedback. All you should do after that is merge your branch with upstream.